SendUserDataProcessingEmailActivity/index.ts

```diff
@@ -0,0 +1,51 @@
+﻿import * as NodeMailer from "nodemailer";
+
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+
+import { MailUpTransport } from "io-functions-commons/dist/src/utils/mailup";
+
+import { getSendUserDataProcessingEmailActivityHandler } from "./handler";
+
+// Whether we're in a production environment
+const isProduction = process.env.NODE_ENV === "production";
+
+// Mailup
+const mailupUsername = getRequiredStringEnv("MAILUP_USERNAME");
+const mailupSecret = getRequiredStringEnv("MAILUP_SECRET");
+
+// Email data
+const EMAIL_TITLE = "IO - Richiesta di Download/Cancellazione Dati Utente";
+const mailFrom = getRequiredStringEnv("MAIL_FROM");
```

Comment:
```suggestion
const mailFrom = getRequiredStringEnv("DPO_EMAIL_FROM_ADDRESS");
```

Reply:
these settings all appear into the app service configuration and it would be difficult to tell what "mail_from" means by looking at the settings : )